docs/user-guide/ui-access.md

```diff
@@ -3,23 +3,25 @@
 
 By default, the Kubernetes Dashboard is deployed as a cluster addon. For 1.2 clusters, it is enabled by default.
 
-If you want to manually install it, visit
+To access the Dashboard, visit
 
 `https://<kubernetes-master>/ui`, which redirects to
 `https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard`.
 
-If you find that you're not able to access the Dashboard, it may be because the
-`kubernetes-dashboard` service has not been started on your cluster. In that case,
-you can start it manually as follows:
+If you find that you're not able to access the Dashboard, you can install and open the latest stable release by running the following commands:
 
 ```shell
-kubectl create -f cluster/addons/dashboard/dashboard-controller.yaml --namespace=kube-system
-kubectl create -f cluster/addons/dashboard/dashboard-service.yaml --namespace=kube-system
+kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml --namespace=kube-system
```

Comment:
Change URL to: https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml